src/main/scala/org/broadinstitute/dsde/workbench/sam/service/TosService.scala

```diff
@@ -63,34 +76,46 @@ class TosService(val directoryDao: DirectoryDAO, val tosConfig: TermsOfServiceCo
     userTos.exists { tos =>
       tos.version.contains(tosConfig.version) && tos.action == TosTable.ACCEPT
     }
+}
 
-  /** Get the terms of service text and send it to the caller
-    * @return
-    *   terms of service text
-    */
-  def getText(file: String, prettyTitle: String): String = {
+trait RemoteDocument {
+  def apply(uri: String)(implicit actorSystem: ActorSystem, executionContext: ExecutionContext): String
+}
+
+object RemoteDocument extends RemoteDocument with LazyLogging {
+  override def apply(uri: String)(implicit actorSystem: ActorSystem, executionContext: ExecutionContext): String =
+    if (uri.startsWith("classpath")) {
```

Comment:
i see why you did this but it feels to me like we should just have this RemoteDocument object only support remote stuff, maybe at least add a comment so as to not confuse a future dev

Reply:
I've renamed this to be clearer.